src/sentry/runner/decorators.py

```diff
@@ -25,3 +34,22 @@ def inner(ctx, *args, **kwargs):
             configure()
         return ctx.invoke(f, *args, **kwargs)
     return update_wrapper(inner, f)
+
+
+def log_level_option(f):
```

Comment:
We should make `log_level_option` itself take a default value to override the default. That way a command may be able to say, `this log level is WARN`, and another could say `I want to be ERROR`, etc.

The API should be: `@log_level_option(default='WARN')`